pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SomaticResultUpload.java

```diff
@@ -256,6 +260,9 @@ private static SomaticResultUpload getSingleSomaticResultUpload(long documentId,
         if (documents.isEmpty()) {
             throw new DSMBadRequestException("Bad request for document with id " + documentId);
         }
+        if (documents.get(0) == null) {
```

Comment:
Added for good measure, just in case something weird happens, to avoid returning null.